Translate/En/21-08.md

```diff
@@ -1,118 +1,121 @@
 
-"The "sword of mind" now ... ... Asuna !?"
+"Asuna...was that the "Sword of Mind" just now...!?"
 
-"Incarnation ... ...?"
+"What's the incarnation...?"
 
-Shake head a little to the word you listen for the first time.
+I shoke my head a little towards the word I heard for the first time.
 
 "No, it is not me."
 
-"Well ...... Renri is ... ...."
+"Well, Renri is..."
 
-As Asuna chased the line of sight of Solterina turning around while saying that, the boy knight who was seriously injured was flying instructions to the troops to intercept the army of the approaching red knight, and he said that he was able to afford to keep Asuna very much I do not think so.
+As Asuna followed Solterina's line of sight as she turns around while saying that, the male knight, who was terribly injured was giving instructions to the troops to intercept the army of the approaching red knight, said he don't think he is able to afford to keep an eye on Asuna any much longer. (??)
 
-However, as long as we are looking for reasons for the phenomenon, we must first defend as many underworld people as possible.
+However, while we are trying to solve this phenomenon, we must first defend as many Underworld people as possible.
 
-Asuna stood up with the help of Solterina, inspiring the exhausted spirit and confirmed the circumstances.
+Asuna stood up with the help of Solterina, inspiring the exhausted spirit, as she assess the circumstances.
 
-And I felt something new despair came into my mind as it turned black and cold water.
+And I felt something new, and full of despair came into my mind, like cold dark water pouring in.
 
-More than 80% of players in China and Korea over ten thousand have already entered battle between ally. Even if it is said, the morale difference, the main faction is overwhelming the other players. Avatar disappearing here and there Blue light pillar of effect rises, every time a ferocious scream echoes.
+More than 80% of players in China and Korea, over ten thousand players have already entered the battle between allies. Even if it is said, the morale difference, the main faction is still overwhelimg the resisting factions. As Avatar's disappear here and there, effects in a pillar of blue light rises, and every time, a ferocious scream echoes.
 
-In addition, some of the main factions ……There is little battle capability left for the Japanese, and the underworld people including the integrity knight Renri also are full of creation. Even if it says that there is an advantage of sacred art and sword skill, I do not think it can be repulsed very much.
+In addition, some of the moderate factions…… There is little battle capability left in the Japanese, and the underworld people including the integrity knight Renri also which are all on their last breath(??). Even if there is an advantage from the possession of sacred art and sword skills, I do not think the opposition can be repulsed much.
```

Comment:
"Creation" = "Destiny" = "_life_ of character". Should be smth like
In addition, not only the moderate/resisting _(choose one)_ faction...little potential to fight is _also_ left in [...] which are _running out of life_.`

Reply:
I'm thinking of changing it to "there is little potential of fight left in" instead, how do you think?